ui/src/components/Dashboard/DocsSidebarList.jsx

```diff
@@ -263,7 +263,9 @@ export default class DocsSidebarList extends Component {
               component={Link}
               to={removeReadmeFromPath(item.path)}
               classes={{ container: classes.listItem }}>
-              <ListItemIcon>{<item.icon />}</ListItemIcon>
+              <ListItemIcon>
+                <item.icon />
+              </ListItemIcon>
```

Comment:
Looks like this was fixed in https://github.com/taskcluster/taskcluster/commit/3da927a303f5e7062780f71bea6648f74626ccb9.